### PR TITLE
Update site (except javadoc) for GWT 2.12.0 release

### PR DESCRIPTION
**Pull request by niloc132**

### CODE REVIEW

File: src/main/markdown/release-notes.md [hunk 1]
```diff
@@ -66,6 +67,56 @@ The GWT Release Notes
...
+- Added missing IOException to Reader.read(...) methods
+- Support ElementType MODULE and RECORD_COMPONENT on annotations
+
+
+### Deprecations and Removals:
+- The `unload` event has been deprecated in browsers, deprecated calls that require it, and rewrite calls that shouldn't need it. Deprecated methods have notes describing how to replace them with more modern strategies.
+- The `mousewheel` event has been deprecated in browsers, its usage in GWT has been replaced with the `wheel` event. This should be transparent to applications.
+- The `com.google.gwt.dev.GetJreEmulation` main class has been deprecated, and is planned to be removed in a future release.
+- The `com.google.gwt.dev.RunWebApp` main class has been deprecated, and is planned to be removed in a future release.
+- The `SppedTracer` classes have been deprecated, and are planned to be removed/replaced in a future release.
```
Comment (FrankHossfeld) [inline src/main/markdown/release-notes.md:102]: Typo: SpeedTracer

**Review by FrankHossfeld — CHANGES_REQUESTED** [review]

**Review by FrankHossfeld — APPROVED** [review]

File: src/main/markdown/gettingstarted-v2.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ the plugin generates a Maven project with separate modules for
 
 ## Prerequisites<a id="prereqs"></a>
 
-1.  GWT 2.11.0 still supports Java 8 (running and building). We suggest to use at least Java 11 or 17. If necessary, download and
+1.  GWT 2.12.0 still supports Java 8 (running and building). We suggest to use at least Java 11 or 17. If necessary, download and
```
Comment (MCMicS) [inline src/main/markdown/gettingstarted-v2.md:26]: I thought Java 8 support dropped with #9899 or I misunderstood?

**Review by MCMicS — APPROVED** [review]

**Review by vegegoku — APPROVED** [review]

File: src/main/markdown/gettingstarted-v2.md (hunk 2 above)
Comment (FrankHossfeld) [reply src/main/markdown/gettingstarted-v2.md:26]: AFAIK, you still need at least Java 11 to run GWT, but your project is able to build and run for Java 8. We had that questions a few days ago. See: https://github.com/gwtproject/gwt/pull/10019#pullrequestreview-2397521908

Comment (MCMicS) [reply src/main/markdown/gettingstarted-v2.md:26]: Thanks for clarifying  👍🏻

**Review by FrankHossfeld — APPROVED** [review]

**Review by andreaskornstaedt — APPROVED** [review]
Looks good. Only (non-blocking) suggestion: Shouldn't -XfragmentCount be marked as a regular (non-experimental) flag now that -XfragmentMerge is marked deprecated ?

**Pull request merged**